src/Backend/Form/Type/EditorType.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Backend\Form\Type;
+
+use Backend\Core\Engine\Model;
+use Backend\Core\Language\Language;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class EditorType extends TextareaType
+{
+    public function configureOptions(OptionsResolver $optionsResolver)
+    {
+        $optionsResolver->setDefaults(['attr' => ['class' => 'inputEditor']]);
+
+        if (!Model::getContainer()->has('header')) {
+            return;
+        }
+        // add the needed javascript to the header;
+        $header = Model::getContainer()->get('header');
```

Comment:
I know, but if it wasn't for that we would have needed another way to add javascript files etc